handlers/server.go

```diff
@@ -53,9 +55,11 @@ func NewWebHookServer() *WebHookServer {
 func AddFlags(fs *pflag.FlagSet, s *WebHookServer) {
 	fs.StringVar(&s.Address, "address", s.Address, "IP address to serve, 0.0.0.0 by default")
 	fs.Int64Var(&s.Port, "port", s.Port, "Port to listen on, 3000 by default")
-	fs.StringVar(&c.Repo, "repo", c.Repo, "refers to the project repo address")
-	fs.StringVar(&c.GitHubToken, "github-token", c.GitHubToken, "contains the githubtoken info")
-	fs.StringVar(&c.WebhookSecret, "webhook-secret", c.WebhookSecret, "contains the webhooksecret key")
+	fs.StringVar(&c.Repo, "repo", c.Repo, "Refers to the project repo address")
+	fs.StringVar(&c.GitHubToken, "github-token", c.GitHubToken, "Contains the githubtoken info")
+	fs.StringVar(&c.WebhookSecret, "webhook-secret", c.WebhookSecret, "Contains the webhooksecret key")
+	fs.StringVar(&c.TravisCIToken, "travis-ci-token", c.TravisCIToken, "Contains Travis-CI access token to trigger the PR build")
+	fs.StringVar(&c.TravisRepoName, "repoName", c.TravisRepoName, "Contains repo name of CI build Ex: kubeedge/kubeedge")
```

Comment:
A quick question, what the difference between c.Repo with c.TravisRepoName? Can we merge them into one?

Reply:
> A quick question, what the difference between c.Repo with c.TravisRepoName? Can we merge them into one?

@m1093782566 
c.Repo -> Git repository (Ex: https://github.com/kubeedge/kubeedge.git)
c.TravisRepoName -> Just a Repo Name (Es: kubeedge/kubeedge)

we cannot merge these 2, as these flags are required for Travis API's

Reply:
Seem wired to me, but Okay for now.